styx-common/src/main/java/com/spotify/styx/util/ParameterUtil.java

```diff
@@ -109,6 +109,18 @@ public static Instant parseDateHour(String dateHour) {
         ISO_LOCAL_DATE_HOUR.parse(dateHour)).atOffset(UTC));
   }
 
+  public static Instant parseDateMonth(String dateMonth) {
+    return Instant.from(LocalDate.from(
+        DateTimeFormatter.ISO_LOCAL_DATE.parse(dateMonth + "-01"))
+        .atStartOfDay(UTC));
```

Comment:
We can reduce duplication by basing this on the other function
```suggestion
    return parseDate(dateMonth + "-01");
```